pkg/cmd/target.go

```diff
@@ -632,9 +632,10 @@ func targetShoot(targetWriter TargetWriter, shoot gardencorev1beta1.Shoot, reade
 
 	// Cache shoot kubeconfig
 	var shootCacheDir string
-	if target.Target[1].Kind == "seed" {
-		shootCacheDir = filepath.Join(pathSeedCache, target.Target[1].Name, shoot.Name)
-	} else if target.Target[1].Kind == "project" {
+	// if target.Target[1].Kind == "seed" {
```

Comment:
in short, this will causing problem for scenario like `gardenctl target -s xxx -t xxx`
There are many combination scenarios in target Stack, the `target.Target[0]` is garden for sure, otherwise code will  error, the `target.Target[1]` could be `seed`, `project`, `shoot` ,`namespace`.  I would suggest you perform full test (especially action related to seed ) against your PR

Reply:
I noticed there are a lot of combination scenarios when using target stack. also mention in another ticket full test required as well, We need to decide the user behavior at the beginning.  

Take the example, the case you mention "-s xxx -t xxx"  I am not understanding here what kind of user case needed and the purpose?  do we have a real case you need target seed then target shoot cluster again?  but definitely there will be more real cases we don't know or need be aware of. 


for my understanding and the design for now

to target seed:  gardenctl target -g xxx --seed soil-gcp --namespace shoot--garden--az-eu2
to target shoot: gardenctl target -g xxx --project garden --shoot az-eu2

Reply:
- "-s xxx -t xxx" is real case, say i want to target a shoot within a seed after target a garden
- there are many other scenarios maybe you are not aware of, e.g
 target shoot
   - gardenctl target shoot xxx (currently stack might be garden/seed , garden/project , garden )
   - gardenctl target xxx (without xxx type specified)

Reply:
still not got the point "-s xxx -t xxx".  to target seed or target shoot are different, that I already provider the command as above which you can also get from the access bar from the gardener console page.  


for the second part about the usage of target shoot whatever you are using target `shoot` or `-t` it is the same goal to target the shoot cluster. Even you mention currently stack might be garden/seed, garden/project, garden, for this part should not take care in `targetShoot` method. that why I suggest remove get seed or the thing you mention garden/seed, garden/project method out of `targetShoot` method.